### DISCUSSION TITLE
Prisma Data Proxy and Planetscale

### DISCUSSION THREAD

**Discussion by jcohenho** [prisma/prisma#12813] [Q&A] [answered]
Hello! I'm building an app using Remix + Prisma +  PlanetScale + Vercel. What are the advantages of creating a cloud prisma project with Planetscale and using Prisma Data Proxy vs not? Planetscale already offers the benefits of scaling thousands of db connections in a serverless environment from what I understand, so I'm curious what the added benefit is of the Data Proxy.

**Comment by nurul3101** [comment, marked as answer]
Hey @jcohenho 👋 ,

Prisma Data Proxy is one of the features of the Prisma Data Platform.
We are working on other features besides the data proxy that we think our users will find very valuable, but right now the data proxy is probably the most relevant and crucial if you are on serverless.

PlanetScale supports a higher number of database connections than other traditional RDBMS providers, 1K on the free plan, 10K in the Scaler plan with the ability to pay for more, so you wouldn't need Prisma Data Proxy as of now, we will be adding many more regions in which Prisma data proxy will run in future so at that time Data Proxy would be useful for you.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
